src/pages/hosts/signup/HostsSignupForm.tsx

```diff
@@ -0,0 +1,302 @@
+    >
+      {({ errors, touched, setFieldValue, submitForm, isSubmitting, values }) => (
+        <Form tag={FormikForm}>
+          <div className="mb-4">
+            <h2 className="h3 text-primary font-weight-normal mb-0">
+              Earn passive income on <span className="font-weight-semi-bold">Beenest</span>
+            </h2>
+          </div>
+
+          <Row className="d-flex flex-sm-column flex-md-row">
+            <Col md={6}>
+              <FormGroup>
+                <Label for={FIRST_NAME} className="form-label">
+                  First Name
+                </Label>
+                <Input
+                  type="text"
+                  name={FIRST_NAME}
+                  id={FIRST_NAME}
+                  tag={Field}
+                  onChange={(event: React.FormEvent<HTMLInputElement>) => handleChange(event, setFieldValue)}
+                  placeholder="First Name"
+                  invalid={!!errors[FIRST_NAME] && touched[FIRST_NAME]}
+                />
+                <FormFeedback>{errors[FIRST_NAME]}</FormFeedback>
+              </FormGroup>
+            </Col>
+            <Col md={6}>
+              <FormGroup>
+                <Label for={LAST_NAME} className="form-label">
+                  First Name
+                </Label>
+                <Input
+                  type="text"
+                  name={LAST_NAME}
+                  id={LAST_NAME}
+                  tag={Field}
+                  onChange={(event: React.FormEvent<HTMLInputElement>) => handleChange(event, setFieldValue)}
+                  placeholder="Last Name"
+                  invalid={!!errors[LAST_NAME] && touched[LAST_NAME]}
+                />
+                <FormFeedback>{errors[LAST_NAME]}</FormFeedback>
+              </FormGroup>
+            </Col>
+          </Row>
+
+          <FormGroup>
+            <Label for={EMAIL} className="form-label">
+              Email Address
+            </Label>
+            <Input
+              type="email"
+              name={EMAIL}
+              id={EMAIL}
+              tag={Field}
+              onChange={(event: React.FormEvent<HTMLInputElement>) => handleChange(event, setFieldValue)}
+              placeholder="Email address"
+              invalid={!!errors[EMAIL] && touched[EMAIL]}
+            />
+            <FormFeedback>{errors[EMAIL]}</FormFeedback>
+          </FormGroup>
+
+          <FormGroup>
+            <Label for={PASSWORD} className="form-label">
+              Password
+            </Label>
+            <Input
+              type="password"
+              name={PASSWORD}
+              id={PASSWORD}
+              tag={Field}
+              onChange={(event: React.FormEvent<HTMLInputElement>) => handleChange(event, setFieldValue)}
+              placeholder="********"
+              invalid={!!errors[PASSWORD] && touched[PASSWORD]}
+            />
+            <FormFeedback>{errors[PASSWORD]}</FormFeedback>
+          </FormGroup>
+
+          <FormGroup>
```

Comment:
do we need to confirm the password? 
just one more step in reducing conversion.

facebook.com doesn't ask for this.

Reply:
we can remove it